validation-test/stdlib/FixedPointConversion.swift.gyb

```diff
@@ -0,0 +1,126 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: %S/../../utils/gyb %s -o %t/FixedPointConversion.swift
+// RUN: %S/../../utils/line-directive %t/FixedPointConversion.swift -- %target-build-swift %t/FixedPointConversion.swift -o %t/a.out_Debug
+// RUN: %S/../../utils/line-directive %t/FixedPointConversion.swift -- %target-build-swift %t/FixedPointConversion.swift -o %t/a.out_Release -O
+//
+// RUN: %S/../../utils/line-directive %t/FixedPointConversion.swift -- %target-run %t/a.out_Debug
+// RUN: %S/../../utils/line-directive %t/FixedPointConversion.swift -- %target-run %t/a.out_Release
+// REQUIRES: executable_test
+
+%{
+
+import gyb
+
+
+
+}%
+
+import StdlibUnittest
+
+var FixedPointConversionTraps = TestSuite("FixedPointToFixedPointConversionTraps")
+var FixedPointConversionFailure = TestSuite("FixedPointToFixedPointConversionFailures")
+
+%{
+
+int_to_int_conversion_template = gyb.parse_template("int_to_int_conversion",
+"""
+%{
+from SwiftIntTypes import all_integer_types
+
+def intMax(bits, signed): 
+    bits = bits - 1 if signed else bits 
+    return (1 << bits) - 1
+    
+def intMin(bits, signed):
+  return -1 * intMax(bits, signed) - 1 if signed else 0
+  
+}%
+
+% for self_ty in all_integer_types(word_bits):
+%   selfBits = self_ty.bits
+%   selfSigned = self_ty.is_signed
+%   selfMin = intMin(selfBits, selfSigned)
+%   selfMax = intMax(selfBits, selfSigned)
+%   Self = self_ty.stdlib_name
+
+%   for other_ty in all_integer_types(word_bits):
+%     otherBits = other_ty.bits
+%     otherSigned = other_ty.is_signed
+%     otherMin = intMin(otherBits, otherSigned)
+%     otherMax = intMax(otherBits, otherSigned)
+%     Other = other_ty.stdlib_name
+
+%     for testValue in [selfMin, selfMax, selfMin - 1, selfMax + 1, otherMin, otherMax]:
+
+%       if testValue < otherMin or testValue > otherMax: 
+%          # Do nothing/continue.
```

Comment:
Could you add a more detailed comment, something like "Can't construct the 'other' value, do nothing."?